docs/integrations/hosts-operating-systems/opentelemetry/linux-opentelemetry.md

```diff
@@ -64,22 +64,35 @@ The following logs, located in your Linux machine's `/var/log` folder, are requi
 - `Messages`
 - `yum.log`
 
-Click on the **Download YAML File** button to get the yaml file.<br/><img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Linux-OpenTelemetry/Linux-YAML.png' alt="YAML" />
-
 :::note 
+By default, the path for Linux log files required for all the distros are pre-populated in the UI. Not all of the files might be available on your Linux distribution. Optionally, you can remove unwanted file paths from the list. OpenTelemetry collection will still work properly even if not all of the files are present on your system. If in doubt, you can leave the default file paths values.
+:::
+
+#### Enable process metric collection (Optional)
+
+By default, the collector will not send process metrics to Sumo Logic; the number of processes running on a host can be very large, which would result in significant increase in Data Points per Minute (DPM).
```

Comment:
```suggestion
By default, the collector will not send process metrics to Sumo Logic. This is because the number of processes running on a host can be very large, which would result in a significant increase in Data Points per Minute (DPM).
```